src/self-model.scm

```diff
@@ -660,12 +662,12 @@
 
 ;; Glance at the last known location of a face that is no longer
 ;; visible.
-(DefineLink
-	(DefinedSchema "glance at lost face")
-	(Put
-		(Evaluation (GroundedPredicateNode "py:glance_at_face")
-			(ListLink (Variable "$face-id")))
-		(DefinedSchema "New departures")))
+;(DefineLink
+;	(DefinedSchema "glance at lost face")
+;	(Put
+;		(Evaluation (GroundedPredicateNode "py:glance_at_face")
+;			(ListLink (Variable "$face-id")))
+;		(DefinedSchema "New departures")))
 
```

Comment:
well, after the above discussion, its clear that removing it is wrong -- it really needs to be restored. She needs to be able to remember where faces have been, and to be able to look in those directions.  

I am kind of irked that you are removing important functions, and doing so without any explanation, and without any code review, and just merging the code.   This causes damage, and it will then take someone else's time to figure out why things don't work, and how to fix them.   Now, the code that we have is ... imperfect. It has a lot of problems and a lot of things that need fixing.  But just cutting this out seems to make things worse, not better...